src/pages/ProtectedRoute/index.js

```diff
@@ -2,7 +2,23 @@ import React from "react";
 import { Navigate, Outlet } from "react-router-dom";
 
 export default function ProtectedRoute() {
-  const isAuthUser = localStorage.getItem("authToken");
+  const getCookie = (cookieName) => {
+    const name = `${cookieName}=`;
+    const decodedCookies = decodeURIComponent(document.cookie);
+    const cookies = decodedCookies.split("; ");
+
+    let matchedCookie = "";
+
+    cookies.forEach((value) => {
+      if (value.indexOf(name) === 0) {
+        matchedCookie = value.substring(name.length);
```

Comment:
```suggestion
        matchedCookie = true;
```
불린값으로써 활용할 의도가 맞을까요?

Reply:
`matchedCookie`가 `getCookie`의 리턴값이고, `isAuthUser`에 들어가서 `true`, `false` 판단시에만 이용하는 것으로 보여서
`Boolean` 타입이 적절할 것 같습니다!

Reply:
넵 반영하였습니다!!